client/my-sites/post-share/index.jsx

```diff
@@ -339,9 +351,24 @@ class PostShare extends Component {
 
 		return (
 			<div className="post-share__services">
-				<h5 className="post-share__services-header">
-					{ translate( 'Connected services' ) }
-				</h5>
+				<SectionHeader className="post-share__services-header" label={ translate( 'Connected accounts' ) }>
+					<Button
+						compact
+						href={ '/sharing/' + siteId }
+						className="post-share__add-button"
+						onMouseEnter={ this.showAddTooltip }
+						onMouseLeave={ this.hideAddTooltip }
+						ref="addAccountButton"
+						aria-label={ translate( 'Add account', { context: 'button label' } ) }>
```

Comment:
I don't remember if the context is still needed. I'm sorry @yoavf but I think I'm asking you this questions twice or more :-(.